src/sentry/static/sentry/app/components/sentryDocumentTitle.tsx

```diff
@@ -0,0 +1,20 @@
+import React, {FunctionComponent} from 'react';
+import DocumentTitle from 'react-document-title';
+
+
+// objSlug argument - project.slug or organization.slug
```

Comment:
If you move this comment above the typescript type, vscode and other tools will show the comment during e.g. autocomplete

Reply:
No worries, it can be simpler:

```

Reply:
thanks )